lib/excerpt_parser.rb

```diff
@@ -114,7 +114,11 @@ def start_element(name, attributes = [])
       @in_quote = !@keep_onebox_source if attributes.include?(%w[class source])
     when "div", "span"
       attributes = Hash[*attributes.flatten]
-      if attributes["class"]&.include?("excerpt")
+
+      # Only match "excerpt" class if it does specifically equal "excerpt
```

Comment:
Post merge pedantic note on a code comment... shouldn't this read: 

> Only match "excerpt" class if it does *not* specifically equal "excerpt hidden" ...

Reply:
Yes! Will fix, thanks!